### DISCUSSION TITLE
Mixed precision

### DISCUSSION THREAD

**Discussion by marian-code** [deepmodeling/deepmd-kit#345] [Q&A]
Hi,

In paper from September 2020 named [Pushing the limit of molecular dynamics with ab initio accuracy to 100 million atoms with machine learning](https://arxiv.org/pdf/2005.00223.pdf) there is talk of mixed precision mode for deepmd-kit. But I haven't found information on how to use this mode. Is it already available in the public version? If so how should I enable it?

**Comment by denghuilu** [comment]
The code is now available on google drive. https://drive.google.com/file/d/1Rk42sTLHETIMD-qAdqTE0mQdM4vaVMJj/view?usp=sharing. One can follow the README files to install the deepmd-kit summit version and run the test models, which include the mix-precision model.

**Reply by marian-code** [reply]
Thank you very much, is there any plan to merge this into this repository to be widely available?
